health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppPreview.js

```
@@ -195,32 +196,42 @@ const renderField = (field, t) => {
+        ],
+        {
+          select: (data) => {
+            const optionsData = _.get(data, `${field?.schemaCode?.split(".")[0]}.${field?.schemaCode?.split(".")[1]}`, []);
+            return optionsData
+              .filter((opt) => (opt?.hasOwnProperty("active") ? opt.active : true))
+              .map((opt) => ({ ...opt, name: `${Digit.Utils.locale.getTransformedLocale(opt.code)}` }));
+          },
+          enabled: field?.isMdms && field?.schemaCode ? true : false,
+        },
+        { schemaCode: "SELCTIONTABMDMSLIST" }
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)